deployment/docker-compose.yml

```diff
@@ -0,0 +1,121 @@
+version: '3.4'
+
+x-defaults: &defaults
+  restart: always
+  image: raiden-services
+  env_file: services.env
+  volumes:
+    - /data/state:/state
+    - ~/keystore:/keystore
+
+services:
+  pfs-ropsten:
+    << : *defaults
+    build: ../
+    environment:
+      - PFS_ETH_RPC=http://geth.ropsten.ethnodes.brainbot.com:8545
+    labels:
+      - "traefik.enable=true"
+      - "traefik.frontend.rule=Host: pfs-ropsten.services-dev.raiden.network"
+
+  pfs-rinkeby:
+    << : *defaults
+    environment:
+      - PFS_ETH_RPC=http://geth.rinkeby.ethnodes.brainbot.com:8545
+    labels:
+      - "traefik.enable=true"
+      - "traefik.frontend.rule=Host: pfs-rinkeby.services-dev.raiden.network"
+
+  pfs-kovan:
+    << : *defaults
+    environment:
+      - PFS_ETH_RPC=http://parity.kovan.ethnodes.brainbot.com:8545
+    labels:
+      - "traefik.enable=true"
+      - "traefik.frontend.rule=Host: pfs-kovan.services-dev.raiden.network"
+
+  # TODO: probably wrong addresses!
+  ms-ropsten:
+    << : *defaults
+    command: ["python3", "-m", "monitoring_service.cli"]
+    environment:
+      - MS_ETH_RPC=http://geth.ropsten.ethnodes.brainbot.com:8545
+      - MS_REGISTRY_ADDRESS=0x5F7d5e025Ac85830a99D60b40e250681012ece1A
+      - MS_MONITOR_CONTRACT_ADDRESS=0xb662B2922B045c1De01Cd0F56742744c7E303bBa
+      - MS_STATE_DB=/state/ms-ropsten.db
+
+  # addresses taken from https://gist.github.com/palango/d28f0313bd60d2608ddebd5ac2db562f
+  ms-rinkeby:
+    << : *defaults
+    command: ["python3", "-m", "monitoring_service.cli"]
+    environment:
+      - MS_ETH_RPC=http://geth.rinkeby.ethnodes.brainbot.com:8545
+      - MS_REGISTRY_ADDRESS=0xE4769320245E4DeEAE5e816C9A827098B2988ef8
+      - MS_MONITOR_CONTRACT_ADDRESS=0xA0ED03b276cA581e8F120b5DcFe152DaAb0162d9
+      - MS_STATE_DB=/state/ms-rinkeby.db
+
+  # TODO: wrong addresses!
```

Comment:
We should do this automatically, but will this even start?

Reply:
Yes, we will only get problems once we do transactions on those contracts.